src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  width: $card-width;
+  background-color: $backgroundColor;
+  border: 1px solid $borderColor;
+  box-sizing: border-box;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
+  &__inner {
+    box-sizing: border-box;
+    padding: 32px 16px 16px;
+  }
+
+  &__img {
+    width: 159px;
+    height: 134px;
+    margin: 0 auto;
+
+    &--imac {
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
+  }
+
+  &__name {
+    width: 166px;
+    margin-top: 40px;
+  }
+
+  &__description {
+    width: 100%;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $darcBlue;
+  }
+
+  &__code {
+    height: 14px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 4px;
+    color: $grey;
+  }
+
+  &__conteiner {
+    box-sizing: content-box;
+    padding: 0;
+    margin: 16px auto 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 24px auto 0;
+  }
+
+  &__priceText {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $grey;
+  }
+
+  &__value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $darcBlue;
+  }
+
+  &__button {
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: $backgroundColor;
+    text-decoration: none;
+    text-transform: uppercase;
+    transition: all 0.5s;
+    border: 1px solid $buttonBorderDefault;
+    box-sizing: border-box;
+  }
+
+  &__buy {
+    height: 40px;
+    width: 166px;
+    box-sizing: border-box;
+    margin: 16px auto 0;
+    background: $blue;
+    border-radius: 5px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: all 0.5s;
+
+    &:hover {
+      box-sizing: border-box;
+      border: 1px solid $blue;
+      background: $backgroundColor;
+
+      & > .card__button {
+        color: $blue;
+      }
```

Comment:
If you delete redundant wrapper near the link, you'll delete these styles and move them to parent class